main.tf

```diff
@@ -112,9 +112,9 @@ module "configure_project" {
   watsonx_project_description = var.watsonx_project_description
   watsonx_project_tags        = var.watsonx_project_tags
   watsonx_mark_as_sensitive   = var.watsonx_mark_as_sensitive
```

Comment:
consider renaming the variable `watsonx_mark_as_sensitive` to `mark_as_sensitive`

Reply:
Done.